AENS.md

```diff
@@ -252,8 +252,8 @@ A client interacting with the blockchain should generate a warning
 if a user tries to claim a name that is not available.
 
 The `pre-claim` has an implicit expiration attached to it. A
-`pre-claim` MUST be considered invalid after 300 blocks, i.e.
-about two days at 10 minute block time.
+`pre-claim` MUST be considered invalid after `floor(60 * 24 * 2/key_block_interval)` blocks, i.e.
```

Comment:
I see this may require change at https://github.com/aeternity/epoch/blob/e9e3b000475b58501fc0daf8e8b1d57ea8b25669/apps/aecore/src/aec_governance.erl#L101